sdk/formrecognizer/ai-form-recognizer/test/public/node/analysis.spec.ts

```diff
@@ -38,9 +45,14 @@ matrix([[true, false]] as const, async (useAad) => {
     let client: DocumentAnalysisClient;
     let recorder: Recorder;
 
-    beforeEach(function (this: Context) {
-      recorder = createRecorder(this);
-      client = new DocumentAnalysisClient(endpoint(), makeCredential(useAad));
+    beforeEach(async function (this: Context) {
+      recorder = await createRecorder(this.currentTest);
+      await recorder.setMatcher("BodilessMatcher");
```

Comment:
Bodiless matcher since the proxy tool complained that the bodies are not matching with the ones in the recording.

Reply:
https://github.com/Azure/azure-sdk-tools/issues/2723 logged - to investigate further.
BodilessMatcher should be fine anyway.

Reply:
I actually support this for FR regardless of whether or not the bodies match. It puts us one step closer to not even recording them in the first place.

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/20266 tracks the work to skip recording the request body